server/routes/contacts/manage/list/listContactsController.ts

```diff
@@ -1,28 +1,31 @@
 import { Request, Response } from 'express'
 import { PageHandler } from '../../../../interfaces/pageHandler'
 import { Page } from '../../../../services/auditService'
-import { PrisonerSearchService, ContactsService } from '../../../../services'
-import PrisonerJourneyParams = journeys.PrisonerJourneyParams
+import { ContactsService } from '../../../../services'
+import ManageContactsJourney = journeys.ManageContactsJourney
 
 export default class ListContactsController implements PageHandler {
-  constructor(
-    private readonly prisonerSearchService: PrisonerSearchService,
-    private readonly contactsService: ContactsService,
-  ) {}
+  constructor(private readonly contactsService: ContactsService) {}
 
   public PAGE_NAME = Page.LIST_CONTACTS_PAGE
 
-  GET = async (req: Request<PrisonerJourneyParams, unknown, unknown>, res: Response): Promise<void> => {
+  GET = async (
+    req: Request<{ prisonerNumber: string; journeyId?: string }, unknown, unknown>,
+    res: Response,
+  ): Promise<void> => {
     const { user, prisonerDetails } = res.locals
     const { journeyId, prisonerNumber } = req.params
-    const journey = req.session.manageContactsJourneys[journeyId]
 
-    journey.prisoner = {
-      firstName: prisonerDetails?.firstName,
-      lastName: prisonerDetails?.lastName,
-      prisonerNumber: prisonerDetails?.prisonerNumber,
-      dateOfBirth: prisonerDetails?.dateOfBirth,
-      prisonName: prisonerDetails?.prisonName,
+    let journey: ManageContactsJourney
+    if (journeyId) {
+      journey = req.session.manageContactsJourneys[journeyId]
+      journey.prisoner = {
+        firstName: prisonerDetails?.firstName,
+        lastName: prisonerDetails?.lastName,
+        prisonerNumber: prisonerDetails?.prisonerNumber,
+        dateOfBirth: prisonerDetails?.dateOfBirth,
+        prisonName: prisonerDetails?.prisonName,
```

Comment:
I believe this is actually unused at this point but I will clean up in the next change as this already got quite large.